src/clojure_lsp/refactor/transform.clj

```diff
@@ -359,7 +359,7 @@
         qualified-ns-to-add (cond
                               (= 1 (count posibilities))
                               (-> posibilities first :alias-ns))]
-    (add-known-libspec zloc ns-to-add qualified-ns-to-add)))
+    (add-known-libspec zloc ns-to-add alias->info)))
```

Comment:
This is causing all of the alias usages to be splatted into the require...

Reply:
Oh, sorry, I accidentally changed this line while creating tests...

Reply:
Fixed it @snoe!